src/views/self/view_teleport.cpp

```diff
@@ -7,21 +7,37 @@
 
 namespace big
 {
+	//You didn't group these originally, but literally everything else that looks like this
+	//is grouped so guess what
+
 	void view::teleport()
 	{
-		ImGui::Text("Blips:");
+		//Did you forget about that helper function you wrote ALREADY?
+		//ImGui::Text("Blips:");
+		components::sub_title("Blips");
```

Comment:
Because a new function is added doesn't mean all old uses of it are updated but yeah I agree.